src/CascadeMinimizer.cc

```diff
@@ -62,6 +62,11 @@ void CascadeMinimizer::setAutoMax(const RooArgSet *pois)
 
 bool CascadeMinimizer::improve(int verbose, bool cascade) 
 {
+    cacheutils::CachingSimNLL *simnllbb = dynamic_cast<cacheutils::CachingSimNLL *>(&nll_);
+    if (simnllbb && runtimedef::get("MINIMIZER_analytic")) {
+      simnllbb->setAnalyticBarlowBeeston(true);
+      minimizer_.reset(new RooMinimizerOpt(nll_));
```

Comment:
Have to reset the minimizer here because the number of const parameters will change after setAnalyticBarlowBeeston(true). In principle the underlying Minuit2Minimizer should be able to handle this, but the way RooMinimizer syncs with it breaks things.